Makefile

```diff
@@ -33,8 +33,12 @@ install-config:
 	@install -m400 pkg/testing-ecdsa.key $(CONFIG_PREFIX)/keys/testing-ecdsa.key
 	@install -m400 pkg/testing-rsa.key $(CONFIG_PREFIX)/keys/testing-rsa.key
 
-$(INSTALL_BIN)/$(NAME): | install-config
-	@GOOS=linux GOARCH=$(ARCH) go build -o $@ ./cmd/$(NAME)/...
+.PHONY: get-gvt
+get-gvt:
+	go get -u github.com/FiloSottile/gvt
```

Comment:
Trying to promote a world where we git clone && git checkout -f <revision>, since `go get` doesn't have much of an integrity story.